WordPress/Classes/WordPress.xcdatamodeld/WordPress 31.xcdatamodel/contents

```diff
@@ -248,6 +248,7 @@
         <attribute name="title" optional="YES" attributeType="String">
             <userInfo/>
         </attribute>
+        <attribute name="videopressGUID" optional="YES" attributeType="String" syncable="YES"/>
```

Comment:
@astralbodies @bummytime this will actually need to be changed because meanwhile @aerych updated the model in the 5.3 release branch. So I will need to merge the release 5.3 and then create a new data model version (32) and update this branch.